driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/nanosleep.bpf.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+#include <helpers/interfaces/variable_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(nanosleep_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+        struct auxiliary_map *auxmap = auxmap__get();
+        if(!auxmap)
+        {
+                return 0;
+        }
+
+        auxmap__preload_event_header(auxmap, PPME_SYSCALL_NANOSLEEP_E);
```

Comment:
Because the event information is known (and not variable), I would suggest to use the ringbuffer in the enter/exit event.

Reply:
agree with @hbrueckner here :)